hack/lib/version.sh

```diff
@@ -71,11 +71,12 @@ kube::version::get_version_vars() {
       # TODO: We continue calling this "git version" because so many
       # downstream consumers are expecting it there.
       DASHES_IN_VERSION=$(echo "${KUBE_GIT_VERSION}" | sed "s/[^-]//g")
+      PLUSES_IN_VERSION=$(echo "${KUBE_GIT_VERSION}" | sed "s/[^+]//g")
       if [[ "${DASHES_IN_VERSION}" == "---" ]] ; then
         # We have distance to subversion (v1.1.0-subversion-1-gCommitHash)
         KUBE_GIT_VERSION=$(echo "${KUBE_GIT_VERSION}" | sed "s/-\([0-9]\{1,\}\)-g\([0-9a-f]\{14\}\)$/.\1\+\2/")
```

Comment:
it looks like this line too would result in an extra `+`

Reply:
In this case the plus is appropriate. If my interpretation of the SemVer is correct you can have a maximum of 3 separators, so if there are 3 dashes, it implies there are no pluses.